docs/pageobjects.md

```diff
@@ -185,8 +185,8 @@ class AttachFile {
 }
 
 // For inheritance
-module.exports = new AttachFile()
-module.exports.AttachFile = AttachFile
+exports.AttachFile = AttachFile
```

Comment:
Why you change it? 
I changed it in https://github.com/Codeception/CodeceptJS/pull/2086 because this export not working.

Reply:
Maybe you must make rebase by master?

Reply:
sorry, my bad. Didn't rebase with master.